includes/admin/post-types/meta-boxes/class.llms.meta.box.certificate.template.sync.php

```diff
@@ -122,7 +122,8 @@ private function sync_action() {
 			$awarded_certificates_number,
 			$awarded_certificate_label
 		);
-		$on_click   = "return confirm('${sync_alert}')";
+		$sync_alert = str_replace( "'", "\'", $sync_alert);
```

Comment:
Using [esc_js()](https://developer.wordpress.org/reference/functions/esc_js/) I believe is more thorough for escaping inline JS